packages/salesforcedx-vscode-core/package.json

```diff
@@ -845,6 +845,11 @@
       "type": "object",
       "title": "%feature_previews_title%",
       "properties": {
+        "salesforcedx-vscode-core.force-push-and-pull.enabled": {
```

Comment:
@smaddox-sf Looking for your feedback on the setting name. Following my comment on how this is a deploy on save feature, should we make the setting in that scope?

e.g.

`salesforcedx-vscode-core.push-or-deploy-on-save.enabled`
`salesforcedx-vscode-core.push-or-deploy-on-save.overrideConflictsOnPush`

Reply:
@brpowell - I like your suggestion and agree we should keep it within the 'upon save' scope.  For the description of this setting, we should also note it's upon save and remove the mention of pull...so something like "Specifies whether to always use --forceoverwrite when doing force:source:push on save".  When ready, Sonal should review the final wording for the setting & description

Reply:
I have used the suggested text and the setting is called `salesforcedx-vscode-core.push-or-deploy-on-save`
Let me know if this needs to change

Reply:
Slight modification -- "Specifies whether to always use --forceoverwrite when you run force:source:push on save"